src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -142,24 +113,22 @@ public void apply() throws ConfigurationException {
 
   @Override
   public void reset() {
-    isModified = false;
-
     final FlutterSdk sdk = FlutterSdk.getGlobalFlutterSdk();
-    final String path = sdk != null ? FileUtilRt.toSystemDependentName(sdk.getHomePath()) : "";
+    final String path = sdk != null ? sdk.getHomePath() : "";
     sdkCombo.getComboBox().getEditor().setItem(path);
 
     updateVersionText();
     updateErrorLabel();
   }
 
   private void updateVersionText() {
-
     final FlutterSdk sdk = FlutterSdk.forPath(getSdkPathText());
     if (sdk == null) {
       versionDetails.setVisible(false);
     }
     else {
       try {
+        final ModalityState modalityState = ModalityState.current();
```

Comment:
Ah, OK.  I guess I thought `updateVersionText()` was getting called on the AWT thread but it's hard to keep straight. 😄 

If I'm understanding you right, isn't it odd that this would ever have worked?


<img width="733" alt="screen shot 2016-10-20 at 5 38 24 am" src="https://cloud.githubusercontent.com/assets/67586/19559996/5e95e1ce-9687-11e6-9dd8-bc409a959259.png">

Why am I not seeing an exception on HEAD?

(Incidentally, these version details are something we don't want to lose from the UI when we make changes to Flutter configuration.  They're very useful for debugging.)